frontend/src/pages/ActivityReport/Pages/topicsResources.js

```diff
@@ -54,7 +54,8 @@ const TopicsResources = ({
             <br />
             Enter the URL for OHS resource(s) used.
             {' '}
-            <a href="https://eclkc.ohs.acf.hhs.gov/">https://eclkc.ohs.acf.hhs.gov/</a>
+            {/* eslint-disable-next-line react/jsx-no-target-blank */}
+            <a target="_blank" rel="noopener" href="https://eclkc.ohs.acf.hhs.gov/">https://eclkc.ohs.acf.hhs.gov/</a>
```

Comment:
1. Consider having the link text be something other than a URL. [WebAIM guidance about URLs in link text](https://webaim.org/techniques/hypertext/link_text#urls).
2. Consider alerting user that link will open in new window. [WebAIM guidance on opening new windows](https://webaim.org/techniques/hypertext/hypertext_links#new_window).